docs/404.html

```diff
@@ -0,0 +1,4518 @@
+
```

Comment:
Is `page` used to generate what's in `content` and then that's used for what's in `docs`?
Is there a reason why we'd need to commit this generated content?